CONTRIBUTORS.yaml

```diff
@@ -0,0 +1,199 @@
+# List of contributors
+#
+# reference maintainers/contributors by their id in tutorial metadata files
+#
+# Institutional sponsors
+#  * Collaborative Research Centre (CRC) 992 - Medical Epigenetics
+#  * de.NBI - German Network for Bioinformatics Infrastructure
+#  * NHGRI - National Human Genome Research Institute at the National Institutes of Health (USA)
+#  * NSF - National Science Foundation (USA)
+#
+---
+abretaud:
+    name: Anthony Bretaudeau
+
+andreasrichter:
+    name: Andreas Richter
+    github: false
+
+annasyme:
+    name: Anna Syme
+
+bebatut:
+    name: Bérénice Batut
+    email: berenice.batut@gmail.com
+
+bernt-matthias:
+    name: Matthias Bernt
+
+bgruening:
+    name: Björn Grüning
+    email: bjoern.gruening@gmail.com
+
+blankclemens:
+    name: Clemens Blank
+    email: blankclemens@gmail.com
+
+daltis20012009:
+    name: Krzys Poterlowicz
+
+dannon:
+    name: Dannon Baker
+
+dpryan79:
+    name: Devon Ryan
+
+dyusuf:
+    name: Dilmurat Yusuf
+
+erasche:
+    name: Eric Rasche
+
+erxleben:
+    name: Anika Exleben
+
+ewran:
+    name: Aisanjiang Wubuli
+
+fidelram:
+    name: Fidel Ramirez
+
+fmareuil:
+    name: Fabien Mareuil
+
+friedue:
+    name: Friederike Dündar
+
+guerler:
+    name: Aysam Guerler
+
+hmenager:
+    name: Hervé Ménager
+
+hrhotz:
+    name: Hans-Rudolf Hotz
+
+jennydg:
+    name: Jennifer Del Giudice
+
+jfallmann:
+    name: Joerg Fallmann
+
+jgoecks:
+    name: Jeremy Goecks
+
+jj-umn:
+    name: James Johnson
+    email: jj@umn.edu
+
+jmchilton:
+    name: John Chilton
+    email: jmchilton@gmail.com
+
+joachimwolff:
+    name: Joachim Wolff
+    email: wolffj@informatik.uni-freiburg.de
+
+lecorguille:
+    name: Gildas Le Corguillé
+
+loraine-gueguen:
+    name: Loraine Guéguen
+
+malloryfreeberg:
+    name: Mallory Freeberg
+
+manabuishii:
+    name: Manabu ISHII
+
+martenson:
+    name: Martin Čech
+    email: marten@bx.psu.edu
+
+moheydarian:
+    name: Mo Heydarian
+
+mwolfien:
+    name: Markus Wolfien
+
+nekrut:
+    name: Anton Nekrutenko
+    email: anton@nekrut.org
+
+nsoranzo:
+    name: Nicola Soranzo
+
+odoppelt:
+    name: Olivia Doppelt-Azeroual
+
+kilpert:
+    name: Fabian Kilpert
+
+pajanne:
+    name: Anne Pajon
+
+polkhe:
+    name: Ekaterina Polkh
+
+pvanheus:
+    name: Peter van Heusden
+
+pratikdjagtap:
+    name: Pratik Jagtap
+    email: pjagtap@umn.edu
+
+pvanvidem:
+    name: Pavankumar Videm
+
+shiltemann:
+    name: Saskia Hiltemann
+    email: zazkia@gmail.com
+
+slegras:
+    name: Stéphanie Legras
+
+slugger70:
+    name: Simon Gladman
+    email: simon.gladman@unimelb.edu.au
+
+stortebecker:
+    name: Florian Christoph Sigloch
+    email: sigloch+github@mailbox.org
+
+tealiie:
```

Comment:
@tealiie since you contributed to this repo I would like to list you here, but I don't know your full name, please let me know if you would like that to be added here :)